packages/binding-coap/src/coap.ts

```diff
@@ -27,14 +27,14 @@ export * from "./coap-client";
 export * from "./coaps-client-factory";
 export * from "./coaps-client";
 
-export class CoapForm extends Form {
-    public "coap:methodCode"?: number; // 1=0.01=GET, 2=0.02=POST, 3=0.03=PUT, 4=0.04=DELETE
-    public "coap:options"?: Array<CoapOption> | CoapOption;
+export class CoapOption {
+    public "cov:optionName": number;
+    public "cov:optionValue": any;
 }
 
-export class CoapOption {
-    public "coap:optionCode": number;
-    public "coap:optionValue": any;
+export class CoapForm extends Form {
+    public "cov:methodName"?: "GET" | "POST" | "PUT" | "DELETE" | "FETCH" | "PATCH" | "iPATCH";
+    public "cov:options"?: Array<CoapOption> | CoapOption;
```

Comment:
First I was confused about the change but in the end you just reorderd CoapOption and CoapForm which causes these changes... might be ok to leave the order so it is more visible "what" has been changed....

Reply:
I was a bit confused myself when I was looking at the this change in the PR :D The rationale behind it was that eslint complained that `CoapForm` was used before it was defined. But I agree, to make things easier to track it would be better if the linting change would be done separately. I'll therefore revert the reordering :)

Reply:
ahh eslint.. I see
I am fine leaving it as is ... no need to shuffle around again! we try to comply to eslint rules anyway...

Reply:
Okay, great :)